frontend/src/views/Waiting.tsx

```diff
@@ -0,0 +1,106 @@
+import React, { useState } from 'react';
+import { useLocation } from 'react-router-dom';
+import { Message } from 'stompjs';
+import { LargeText, UserNicknameText } from '../components/core/Text';
+import {
+  addUser, SUBSCRIBE_URL, connect,
+  deleteUser, SOCKET_ENDPOINT, subscribe, User,
+} from '../api/Socket';
+import ErrorMessage from '../components/core/Error';
+
+type WaitingRoomPageProps = {
```

Comment:
nit: Maybe rename this to something involving Location, as I think it's being passed to useLocation as opposed to the component props

Reply:
Done -- updated to `LobbyPageLocation` 👍